solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -922,20 +920,15 @@ private UpdateHandler createUpdateHandler(String className, UpdateHandler update
     return createReloadedUpdateHandler(className, "Update Handler", updateHandler);
   }
 
-  public SolrCore(CoreContainer coreContainer, CoreDescriptor cd, ConfigSet configSet) {
-    this(coreContainer, cd, configSet, null,
-        null, null, null, false);
-  }
-
   public CoreContainer getCoreContainer() {
     return coreContainer;
   }
 
+  SolrCore(CoreContainer coreContainer, CoreDescriptor cd, ConfigSet configSet) {
```

Comment:
I just co-located the constructors together, and furthermore reduced the access scope.  I could have left this out.  The former javadocs were obsoleted recently.